scripts/check_lint.sh

```diff
@@ -16,6 +16,9 @@ if [ ! $(command -v ${DOCKER_CMD}) ]; then
     exit 0
 fi
 
+echo "Linting wallet-sdk-js..."
```

Comment:
For consistency, I would expect the top-level module to get linted first, followed by the submodules (so I'd move this after the top-level module lint)

Reply:
Yes sure